docker_images/peft/app/pipelines/text_generation.py

```diff
@@ -46,7 +46,7 @@ def __call__(self, inputs: str, **kwargs) -> str:
                 resp = self._process_req(inputs, **kwargs)
         else:
             resp = self._process_req(inputs, **kwargs)
-        return resp
+        return {"generated_text": resp[0]}
```

Comment:
I *think* it shoudl be `[{"generated_text": ..}]` no ?

Reply:
Addressed this now